doc/api/stream.md

```diff
@@ -1382,6 +1385,12 @@ constructor and implement the `writable._write()` method. The
 `writable._writev()` method *may* also be implemented.
 
 #### Constructor: new stream.Writable([options])
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/18438
+    description: add emitClose option to specify if close is emitted on destroy
```

Comment:
Tiny nit: I think so far all of these desciprions are capitalized & punctuated, and it would be nice to keep in line with that. Also, putting `close` in a code block via backticks might be nice, to distinguish it as an event name.